### DISCUSSION TITLE
/sending-mails-embedded-images-via-microsoft-graph/

### DISCUSSION THREAD

**Discussion by comment-migrator** [estruyf/eliostruyf-comments#216] [Comments]
Sending mails with embedded images via the Microsoft Graph
        
[https://www.eliostruyf.com/sending-mails-embedded-images-via-microsoft-graph/](https://www.eliostruyf.com/sending-mails-embedded-images-via-microsoft-graph/)

**Comment by comment-migrator** [comment]
```header
Originally posted by Jaded Potato on 2017-02-02 18:44
```

Absolutely brilliant! This is exactly what I was looking for.

**Comment by comment-migrator** [comment]
```header
Originally posted by codewriter93 on 2019-06-19 15:07
```

Is the maximum attachment size still 4MB? If so, how do we work around it?

**Comment by comment-migrator** [comment]
```header
Originally posted by caradri3 on 2021-04-08 13:51
```

what is the accessToken?

**Comment by CraigChamberlain** [comment]
Please consider adding the following property to your attachment object so that when you inline an image it's hidden as a downloadable attachment:

    isInline = $true

https://learn.microsoft.com/en-us/graph/api/resources/attachment?view=graph-rest-1.0#properties

**Reply by estruyf** [reply]
Thanks @CraigChamberlain, I'll update this in the post.
